### PR TITLE
Adding physics-tenpy

### PR DESCRIPTION
**Pull request by jhauschild**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jhauschild** [comment]
Ready for review.

It's a python package with compiled C parts stemming from Cython.
The compiled parts include `numpy` and `scipy.linalg.cython_blas`.
This is my first contribution to conda-forge, so I'm not 100% sure if I got all the requirements right, but I tested that it works locally (on linux).

**Comment by jhauschild** [comment]
Ready for review.

It's a python package with compiled C parts (from Cython, importing numpy and `scipy.linalg.cython_blas`, in case that's relevant).
I hope I got all the requirements right (first time contribution here); I tested it locally on my machine and it seemed to work.

File: recipes/physics-tenpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - cython >=0.29
+    - numpy >=1.13
+    - scipy >=0.18
```
Comment (chrisburr) [inline recipes/physics-tenpy/meta.yaml:25]: Is scipy needed in host? Normally packages which are used from python are only needed in run.

Comment (jhauschild) [reply recipes/physics-tenpy/meta.yaml:25]: Yes, it's needed; I quickly tried it and the build fails if scipy is removed.
The reason is that there is a cimport from `scipy.linalg.cython_blas` in the cython file.

**Comment by jhauschild** [comment]
@chrisburr I don't want to be annoying, just checking if there is anything else to be done from my side?

**Pull request merged**